website/docs/d/outputs.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_outputs"
+sidebar_current: "docs-datasource-tfe-state-outputs"
+description: |-
+  Get output values from another organization/workspace.
+---
+# Data Source: tfe_outputs
+
+This data source is used to retrieve the state outputs for a given workspace.
+It enables output values in one Terraform configuration to be used in another.
+
+Any output value that is marked as sensitive is not hidden.
```

Comment:
This is rather whiplashing without the added context that the _other_ configuration's sensitivity context is not preserved 😆 How about:

```suggestion
Note that any output value marked as sensitive from the remote workspace configuration is not redacted in the _current_ configuration when using this datasource. It is the responsibility of the configuration using this datasource to redetermine sensitive values (e.g. using [sensitive()](https://www.terraform.io/docs/language/functions/sensitive.html) or redeclaring a sensitive output value).
```